app/src/main/java/com/gatchimath/totask/SqlStorage.java

```diff
@@ -0,0 +1,88 @@
+package com.gatchimath.totask;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import java.util.ArrayList;
+
+/**
+ * Represents all task-related SQL storage.
+ * <p>
+ * Most of the methods here are basically just SQLite commands.
+ * This class should only be instantiated in a
+ * {@link com.gatchimath.totask.TaskStorage}.
+ *
+ * @see com.gatchimath.totask.TaskTableEntry,
+ *      com.gatchimath.totask.TaskDbHelper
+ */
+public class SqlStorage {
```

Comment:
This class is complex enough to require unit tests. You should be able to get a test context using the instrumentation library, which will allow you to set up the appropriate Context to hand to SqlStorage.

https://developer.android.com/topic/libraries/testing-support-library/index.html